flexmeasures/data/migrations/versions/ea84cefba5c0_added_logo_url_field_to_account_model.py

```diff
@@ -0,0 +1,31 @@
+"""added logo_url field to account model
+
+Revision ID: ea84cefba5c0
+Revises: 4b5aa7856932
+Create Date: 2024-08-19 14:07:14.187428
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "ea84cefba5c0"
+down_revision = "4b5aa7856932"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Please check the commands and remove the comments.

Reply:
sure 👍